public/docs/ts/latest/guide/style-guide.jade

```diff
@@ -236,24 +239,20 @@ a(href="#toc") Back to top
 
 .s-rule.do
   :marked
-    **Do** use upper camel case for class names. Match the name of the symbol to the name of the file.
+    **Do** match the name of the symbol to the name of the file. 
```

Comment:
Should these be two separate DOs?
**Do** use upper camel case for class names.
**Do** match the name of the symbol to the name of the file.